cmd/expired-authz-purger/main.go

```diff
@@ -0,0 +1,165 @@
+// Copyright 2014 ISRG.  All rights reserved
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package main
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/cactus/go-statsd-client/statsd"
+	"github.com/codegangsta/cli"
+	"github.com/jmhodges/clock"
+	"gopkg.in/gorp.v1"
+
+	"github.com/letsencrypt/boulder/cmd"
+	blog "github.com/letsencrypt/boulder/log"
+	"github.com/letsencrypt/boulder/metrics"
+	"github.com/letsencrypt/boulder/sa"
+)
+
+type expiredAuthzPurger struct {
+	stats statsd.Statter
+	log   blog.Logger
+	clk   clock.Clock
+	db    *gorp.DbMap
+
+	batchSize int64
+}
+
+func (p *expiredAuthzPurger) purgeAuthzs(purgeBefore time.Time, yes bool) (int64, error) {
+	if !yes {
+		var count int
+		err := p.db.SelectOne(&count, `SELECT COUNT(pa.id) FROM pendingAuthorizations AS pa WHERE expires <= ?`, purgeBefore)
```

Comment:
I gave some bad advice early in the project claiming `COUNT(1)` or `COUNT(field)` was faster than `COUNT(*)`. Later research seems to indicate that's not true - they should all have the same performance. However, I'd like to consolidate on one form. Let's pick `COUNT(1)` since it's currently more prevalent.